### ISSUE TITLE
GSLB_Domain missing the persistence option 

### ISSUE THREAD

**Issue by fatpelt** [F5Networks/f5-appsvcs-extension#566]
Not sure if this is a bug or a feature request:

### Environment
 * Application Services Version: 3.33.0
 * BIG-IP Version: 15.1.5

### Summary
There is no option in AS3 to mimic the following tmsh command line:
create wideip a wip.domain.com enabled metadata none persistence enabled persist-cidr-ipv4 32 persist-cidr-ipv6 128 pools add {gpool_poolName {app-service none order 0 ratio 1}}

specifically the persistence options

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/refguide/schemaref/GSLB_Domain.schema.json.html

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "schemaVersion": "3.14.0",
    "class": "ADC",
    "Tenant": {
        "class": "Tenant",
        "Application": {
            "template": "generic",
            "wip.domain.com": {
                "class": "GSLB_Domain",
                "domainName": "wip.domain.com",
                "enabled": true,
                "resourceRecordType": "A",
                "pools": [
                    { "use": "gpool_poolName" }
                ],
                "poolLbMode": "round-robin",
                "persistence": true
            },
        }
    }
}
```

no error is thrown

### Expected Behavior
persistence should be enabled

### Actual Behavior
persistence is not enabled

**Comment by dstokesf5** [comment]
Thank you for your feedback. I am changing this over to a feature request.

**Comment by DebasishBahinipati** [comment]
Any updates or timelines on this request, please let us know.

**Comment by mdditt2000** [comment]
Created Jira AUTOTOOL-3031 for this issue. Customer identified and prioritized.

**Comment by sunitharonan** [comment]
This has been resolved in the 3.41.0 release of AS3.
